gbus/messages.go

```diff
@@ -81,6 +88,10 @@ func (bm *BusMessage) SetPayload(payload Message) {
 	bm.Payload = payload
 }
 
+func (bm *BusMessage) SetPayloadBody(payload []byte) {
```

Comment:
What is the use case for such a scenario?

Reply:
we want to send avro events from go services, and we are using gobSerialization because the avro event register in eventcollector.
so we need to send the body as []byte without gob encoding

Reply:
@hadascoh first you should be using the Prorobuf serializer and not gob for your services...that is the standard wire format we are using in BP  :)
Second,  are these messages sent from the publisher service you have written or from another business service ?

Reply:
after we develope the "bp_event_publisher" that catch avro events and send them to event collector we wanted to test the flow from other services than spaceman.

so I need to send Avro events and not Proto from my service ( bp_account_risk_assessment)

Reply:
@hadascoh you're in luck !! as grabbit supports Avro serialization thanks to @vladshub  :)

[https://github.com/wework/grabbit/blob/master/docs/SERIALIZATION.md](https://github.com/wework/grabbit/blob/master/docs/SERIALIZATION.md)

[https://github.com/wework/grabbit/blob/master/gbus/serialization/avro.go](https://github.com/wework/grabbit/blob/master/gbus/serialization/avro.go)